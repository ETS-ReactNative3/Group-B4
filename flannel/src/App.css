* {
  margin: 0;
  padding: 0;
}

body {
}

.form {
  max-width: 330px;
  margin: 0 auto;
  display: flex;
  flex-direction: column;
  background: white;
  padding: 20px;
  margin-top: 30px;
}

