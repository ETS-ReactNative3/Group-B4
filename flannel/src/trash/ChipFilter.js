/* eslint-disable react/jsx-key */
/* eslint-disable react/prop-types */
import React, {useState,useEffect} from 'react';
import TextField from '@mui/material/TextField';
import Autocomplete from '@mui/material/Autocomplete';
import { Box, Chip} from '@mui/material'
import DeleteIcon from '@mui/icons-material/Delete';
//import CustomChip from '../components/CustomChip'


export default function ChipFilter(props){
    const classList = [
        {label:"CS130"},
        {label:"CS118"},
        {label:"CS131"},
        {label:"CS111"}
    ];
    const affiliationsList = [
        {label: "DevX"},
        {label: "Bruin Racing"},
        {label: "ACM"},
        {label: "UPE"}
    ];
    const interestList = [
        {label: "Hiking"},
        {label: "Swimming"},
        {label: "Gymming"},
        {label: "Walking"}
    ];
    const [classSetList,setClassSetList] = useState([]);
    const [affiliationSetList,setAffiliationSetList] =useState([]);
    const [interestSetList,setInterestSetList] =useState([]);
    const [itemList,setList] = useState([]);
    const classDelete = (label) => () => {
        const newList = classSetList.filter((item) =>  item.item.label !== label);
        setClassSetList(newList);
    }
    const affiliationDelete = (label) => () => {
        const newList = affiliationSetList.filter((item) =>  item.item.label !== label);
        setAffiliationSetList(newList);
    }
    const interestDelete = (label) => () => {
        const newList = interestSetList.filter((item) =>  item.item.label !== label);
        setInterestSetList(newList);
    }
    useEffect(() =>{
        props.changeClasses(classSetList);
        props.changeAffiliations(affiliationSetList);
        props.changeInterests(interestSetList);
    })
    return(
    <div style = {{display:"flex",flexDirection:"column"}}>
        <div style={{padding:"20px"}}>
            <h1>Classes</h1>
            <Autocomplete
                disablePortal
                id="combo-box-demo"
                options={classList}
                sx={{ width: 300 }}
                onChange = {(_event,item) => {
                    let found = false;
                    classSetList.map((e) => {
                        if (e.item.label === item.label){
                            found = true;
                        }
                    })
                    if (!found){
                        const newList = classSetList.concat({item});
                        setClassSetList(newList);
                    }
                }}
                renderInput={(params) => <TextField {...params} label={props.type} />}
            />
            <Box sx = {{display:"flex",flexWrap:"wrap", maxWidth: 300}}> 
                {classSetList.map((item) =>(
                    <div>
                        <Chip sx = {{borderRadius:3.5, backgroundColor: '#A4C3D2',color:"grey"}} label={item.item.label} key={item.item.label} deleteIcon = {<DeleteIcon/>} onDelete = {classDelete(item.item.label)}/>
                    </div>
                ))}
            </Box>
        </div>
        <div style={{padding:"20px"}}>
            <h1>Affiliations</h1>
            <Autocomplete
                disablePortal
                id="combo-box-demo"
                options={affiliationsList}
                sx={{ width: 300 }}
                onChange = {(_event,item) => {
                    let found = false;
                    affiliationSetList.map((e) => {
                        if (e.item.label === item.label){
                            found = true;
                        }
                    })
                    if (!found){
                        const newList = affiliationSetList.concat({item});
                        setAffiliationSetList(newList);
                    }
                }}
                renderInput={(params) => <TextField {...params} label={props.type} />}
            />
            <Box sx = {{display:"flex",flexWrap:"wrap", maxWidth: 300}}> 
                {affiliationSetList.map((item) =>(
                    <div>
                        <Chip sx = {{borderRadius:3.5, backgroundColor: '#A4C3D2',color:"grey"}} label={item.item.label} key={item.item.label} deleteIcon = {<DeleteIcon/>} onDelete = {affiliationDelete(item.item.label)}/>
                    </div>
                ))}
            </Box>
        </div>
        <div style={{padding:"20px"}}>
            <h1>Interests</h1>
            <Autocomplete
                disablePortal
                id="combo-box-demo"
                options={interestList}
                sx={{ width: 300 }}
                onChange = {(_event,item) => {
                    let found = false;
                    interestSetList.map((e) => {
                        if (e.item.label === item.label){
                            found = true;
                        }
                    })
                    if (!found){
                        const newList = interestSetList.concat({item});
                        setInterestSetList(newList);
                    }
                }}
                renderInput={(params) => <TextField {...params} label={props.type} />}
            />
            <Box sx = {{display:"flex",flexWrap:"wrap", maxWidth: 300}}> 
                {interestSetList.map((item) =>(
                    <div>
                        <Chip sx = {{borderRadius:3.5, backgroundColor: '#A4C3D2',color:"grey"}} label={item.item.label} key={item.item.label} deleteIcon = {<DeleteIcon/>} onDelete = {interestDelete(item.item.label)}/>
                    </div>
                ))}
            </Box>
        </div>
      </div>
    )
}