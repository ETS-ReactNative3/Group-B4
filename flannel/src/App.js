import React from 'react';
import LogIn from './pages/LogIn';
import SignUp from './pages/SignUp';
import Chat from './Chat.js'
import {Route,Routes} from 'react-router-dom';
import './App.css'

import logo from './logo.svg';
import './App.css';

function App() {
  return (
      <Routes>
        <Route exact path = "/" element = {<LogIn/>} />
        <Route exact path = "/SignUp" element = {<SignUp/>} />
        <Route exact path = "/Chat" element = {<Chat sender={"user1"} reciever={"user2"}/>} />
      </Routes>
  );
}

export default App;