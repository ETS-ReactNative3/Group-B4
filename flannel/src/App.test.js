import React from 'react'
import ReactDOM from 'react-dom'
import App from './App'
import ChipFilter from '../src/components/ChipFilter'
import UserCard from '../src/components/Explore/UserCard'
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';

describe('Testing React Functional Components', () => {
    // Smoke test to mount App and ensure it doesn't throw during rendering
    it('App renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<App />, div)
    })
    
    it('Chip Filter Component Render Check', () => {
        function dummyHook(){
            //not checking hook useState, simply checking for render errors
        }
        const div = document.createElement('div');
        ReactDOM.render(
            <ChipFilter
            setTagOptions={dummyHook}
            type="Affiliations"
            tagOptions={['a','b','c','d','e','f','g','h']}
            defaultShownTags={['a','b','c','d']}
            setSelectedTags={dummyHook}
            selectedTags={['a','b']}
        /> 
        ,
        div)
    })
    it ('User Card Component Render Check', () => {
        const info = {
            index: 1,
            username:"testing",
            year: "testing",
            major:"testing",
            pronouns:"testing",
            classTags: [],
            interestTags:[],
            affiliationTags:[],
            bio:"testing"
        }
        const div = document.createElement('div');
        ReactDOM.render(
            <UserCard key={info.index}
            displayName={info.username}
            year={info.year}
            major={info.major}
            pronouns={info.pronouns}
            classTags={info.classTags}
            interestTags={info.interestTags}
            affiliationTags={info.affiliationTags}
            bio={info.bio}
            />,
            div
        )
    })

    it('App Snapshot Check',() => {
        const tree = renderer.create(<App></App>).toJSON();
        expect(tree).toMatchSnapshot();
    })

    it('Chip Filter Check',() => {
        function dummyHook(){
            //not checking hook useState, simply checking for render errors
        }
        const tree = renderer.create(            
        <ChipFilter
            setTagOptions={dummyHook}
            type="Affiliations"
            tagOptions={['a','b','c','d','e','f','g','h']}
            defaultShownTags={['a','b','c','d']}
            setSelectedTags={dummyHook}
            selectedTags={['a','b']}
        />).toJSON();
        expect(tree).toMatchSnapshot();
    })

    it ('User Card Check', () => {
        const info = {
            index: 1,
            username:"testing",
            year: "testing",
            major:"testing",
            pronouns:"testing",
            classTags: [],
            interestTags:[],
            affiliationTags:[],
            bio:"testing"
        }
        const tree = renderer.create(
            <UserCard key={info.index}
            displayName={info.username}
            year={info.year}
            major={info.major}
            pronouns={info.pronouns}
            classTags={info.classTags}
            interestTags={info.interestTags}
            affiliationTags={info.affiliationTags}
            bio={info.bio}
            />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })

    it ('User Card Field Population Check',() => {
        const info = {
            index: 1,
            username:"Brandon",
            year: "Senior",
            major:"CS",
            pronouns:"He/Him",
            classTags: [],
            interestTags:[],
            affiliationTags:[],
            bio:"Bio"
        }
        const { getByTestId } = render(<UserCard 
            key={info.index}
            displayName={info.username}
            year={info.year}
            major={info.major}
            pronouns={info.pronouns}
            classTags={info.classTags}
            interestTags={info.interestTags}
            affiliationTags={info.affiliationTags}
            bio={info.bio}
        />)
        expect(getByTestId("username_test")).toHaveTextContent("Brandon");
        expect(getByTestId("year_test")).toHaveTextContent("Senior");
        expect(getByTestId("major_test")).toHaveTextContent("CS");
        expect(getByTestId("pronoun_test")).toHaveTextContent("He/Him");
        expect(getByTestId("bio_test")).toHaveTextContent("Bio");
        //fields have been set in the constant info from above
    })
})