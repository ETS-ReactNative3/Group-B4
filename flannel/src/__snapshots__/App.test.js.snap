// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing React Functional Components App Snapshot Check 1`] = `
<div
  className="MuiBox-root css-14xi6cv"
>
  <div
    className="MuiBox-root css-1085s9"
  >
    <div
      className="MuiBox-root css-19z0rgy"
    >
      <img
        alt="Logo"
        src="bearLogo.png"
      />
      <p
        className="MuiTypography-root MuiTypography-body1 css-1kvtn7e-MuiTypography-root"
      >
        FLANNEL
      </p>
    </div>
  </div>
  <div
    className="MuiBox-root css-5v3sv9"
  >
    <form
      onSubmit={[Function]}
      style={
        Object {
          "display": "flex",
          "flexDirection": "column",
        }
      }
    >
      <h1
        style={
          Object {
            "fontFamily": "Work Sans",
            "fontSize": 40,
            "fontWeight": "bold",
            "textAlign": "center",
          }
        }
      >
        Sign In
      </h1>
      <div
        className="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
        style={
          Object {
            "padding": "5px",
          }
        }
      >
        <label
          className="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary Mui-required css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink={false}
          htmlFor="username"
          id="username-label"
        >
          Username
          <span
            aria-hidden={true}
            className="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
          >
             
            *
          </span>
        </label>
        <div
          className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
            disabled={false}
            id="username"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            type="text"
          />
          <fieldset
            aria-hidden={true}
            className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              className="css-173wfxe"
            >
              <span>
                Username
                 
                *
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
        style={
          Object {
            "padding": "5px",
          }
        }
      >
        <label
          className="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary Mui-required css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink={false}
          htmlFor="password"
          id="password-label"
        >
          Password
          <span
            aria-hidden={true}
            className="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
          >
             
            *
          </span>
        </label>
        <div
          className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
            disabled={false}
            id="password"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            type="password"
          />
          <fieldset
            aria-hidden={true}
            className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              className="css-173wfxe"
            >
              <span>
                Password
                 
                *
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <button
        className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "padding": "10px",
          }
        }
        tabIndex={0}
        type="submit"
      >
        Sign In!
      </button>
      <p
        className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
      >
        <a
          href="/SignUp"
          onClick={[Function]}
        >
          Create an Account
        </a>
      </p>
    </form>
  </div>
</div>
`;

exports[`Testing React Functional Components Chip Filter Check 1`] = `
<div
  className="MuiBox-root css-1kqyu2f"
>
  <div
    aria-expanded={false}
    aria-owns={null}
    className="MuiAutocomplete-root MuiAutocomplete-hasPopupIcon css-16awh2u-MuiAutocomplete-root"
    onClick={[Function]}
    onKeyDown={[Function]}
    onMouseDown={[Function]}
    role="combobox"
  >
    <div
      className="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
      style={
        Object {
          "width": 250,
        }
      }
    >
      <label
        className="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
        data-shrink={false}
        htmlFor="combo-box-demo"
        id="combo-box-demo-label"
      >
        Affiliations
      </label>
      <div
        className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-154xyx0-MuiInputBase-root-MuiOutlinedInput-root"
        onClick={[Function]}
      >
        <input
          aria-activedescendant={null}
          aria-autocomplete="list"
          aria-invalid={false}
          autoCapitalize="none"
          autoComplete="off"
          autoFocus={false}
          className="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
          disabled={false}
          id="combo-box-demo"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onMouseDown={[Function]}
          readOnly={false}
          required={false}
          spellCheck="false"
          type="text"
          value=""
        />
        <div
          className="MuiAutocomplete-endAdornment css-1q60rmi-MuiAutocomplete-endAdornment"
        >
          <button
            aria-label="Open"
            className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-popupIndicator css-qzbt6i-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-popupIndicator"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={-1}
            title="Open"
            type="button"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="ArrowDropDownIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
            <span
              className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <fieldset
          aria-hidden={true}
          className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            className="css-173wfxe"
          >
            <span>
              Affiliations
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
  </div>
  <div
    className="MuiBox-root css-1262k1n"
  >
    <div
      className="MuiButtonBase-root MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-deletable MuiChip-deletableColorDefault MuiChip-filledDefault css-rr4eyv-MuiButtonBase-root-MuiChip-root"
      onBlur={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        className="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
      >
        a
      </span>
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-deleteIcon MuiChip-deleteIconMedium MuiChip-deleteIconColorDefault MuiChip-deleteIconOutlinedColorDefault css-i4bv87-MuiSvgIcon-root"
        data-testid="CloseIcon"
        focusable="false"
        onClick={[Function]}
        viewBox="0 0 24 24"
      >
        <path
          d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
        />
      </svg>
    </div>
    <div
      className="MuiButtonBase-root MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-deletable MuiChip-deletableColorDefault MuiChip-filledDefault css-rr4eyv-MuiButtonBase-root-MuiChip-root"
      onBlur={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        className="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
      >
        b
      </span>
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-deleteIcon MuiChip-deleteIconMedium MuiChip-deleteIconColorDefault MuiChip-deleteIconOutlinedColorDefault css-i4bv87-MuiSvgIcon-root"
        data-testid="CloseIcon"
        focusable="false"
        onClick={[Function]}
        viewBox="0 0 24 24"
      >
        <path
          d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
        />
      </svg>
    </div>
  </div>
</div>
`;

exports[`Testing React Functional Components User Card Check 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1fjn7sr-MuiPaper-root-MuiCard-root"
>
  <div
    className="MuiBox-root css-u4p24i"
  >
    <div
      className="MuiBox-root css-dvxtzn"
    >
      <img
        alt="Profile Picture"
        className="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-1eil56s-MuiCardMedia-root"
        src="ProfilePicture.jpeg"
      />
      <div
        className="MuiBox-root css-1gb9akv"
      >
        <p
          className="MuiTypography-root MuiTypography-body1 css-bk7kvn-MuiTypography-root"
          data-testid="username_test"
        >
          testing
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 css-1pshicg-MuiTypography-root"
          data-testid="pronoun_test"
        >
            
          testing
        </p>
      </div>
      <div
        className="MuiBox-root css-u4p24i"
      >
        <p
          className="MuiTypography-root MuiTypography-body1 css-14zr62y-MuiTypography-root"
          data-testid="year_test"
        >
          testing
           Year
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 css-14zr62y-MuiTypography-root"
          data-testid="major_test"
        >
           
          testing
        </p>
      </div>
    </div>
    <div
      className="MuiBox-root css-1xhj18k"
    >
      <div
        className="MuiBox-root css-rjeo72"
      />
      <div
        className="MuiBox-root css-rjeo72"
      />
      <div
        className="MuiBox-root css-rjeo72"
      />
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-ik7bkm-MuiButtonBase-root-MuiIconButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
        data-testid="ExpandMoreIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
        />
      </svg>
    </button>
  </div>
  <div
    className="MuiBox-root css-gg4vpm"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 css-ome9rp-MuiTypography-root"
      data-testid="bio_test"
    >
      testing
    </p>
    <div
      className="MuiBox-root css-u43kzn"
    >
      <button
        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-ik7bkm-MuiButtonBase-root-MuiIconButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="EmailIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4-8 5-8-5V6l8 5 8-5v2z"
          />
        </svg>
      </button>
      <button
        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-ik7bkm-MuiButtonBase-root-MuiIconButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="InstagramIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M7.8 2h8.4C19.4 2 22 4.6 22 7.8v8.4a5.8 5.8 0 0 1-5.8 5.8H7.8C4.6 22 2 19.4 2 16.2V7.8A5.8 5.8 0 0 1 7.8 2m-.2 2A3.6 3.6 0 0 0 4 7.6v8.8C4 18.39 5.61 20 7.6 20h8.8a3.6 3.6 0 0 0 3.6-3.6V7.6C20 5.61 18.39 4 16.4 4H7.6m9.65 1.5a1.25 1.25 0 0 1 1.25 1.25A1.25 1.25 0 0 1 17.25 8 1.25 1.25 0 0 1 16 6.75a1.25 1.25 0 0 1 1.25-1.25M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3z"
          />
        </svg>
      </button>
      <button
        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-ik7bkm-MuiButtonBase-root-MuiIconButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="FacebookIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
          />
        </svg>
      </button>
      <button
        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-ik7bkm-MuiButtonBase-root-MuiIconButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="TwitterIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
          />
        </svg>
      </button>
      <button
        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-ik7bkm-MuiButtonBase-root-MuiIconButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="LinkedInIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
          />
        </svg>
      </button>
    </div>
  </div>
</div>
`;
